packages/jaspr/lib/src/ui/elements/div.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:jaspr/src/ui/elements/base.dart';
+
+class DivElement extends BaseElement {
+  const DivElement({
+    super.key,
+    super.id,
+    super.style,
+    super.classes,
+    super.attributes,
+    super.events,
+    super.child,
+    super.children,
+  }) : super(tag: 'div');
+}
```

Comment:
I think these wrapper elements are then not needed when the html pr gets merged. 

We should only keep / implement the components that provide value beyond the raw html tag, like `Image`, `Table`, `ListView` etc.

Reply:
Yes but right now I have a lot of usages of this element in `jaspr/bootstrap` library. So I only mark this as a deprecated and after merge of `jaspr/html` I will update `jaspr/bootstrap` and this element I will remove then.

Reply:
Makes sense